### PR TITLE
Remove loan from cart

### PR DESCRIPTION
**Pull request by ChrisCenatie**

### CODE REVIEW

File: app/views/cart_items/index.html.erb [hunk 1]
```diff
@@ -1,21 +1,31 @@
+<<<<<<< HEAD
```
Comment (russelleh) [inline app/views/cart_items/index.html.erb]: This should be removed

**Comment by russelleh** [comment]
There's a merge thing at the first line of the cart items index view...

**Comment by russelleh** [comment]
Other than that, looks good to me! Nice merge conflict resolution skills. ☮

**Comment by ChrisCenatie** [comment]
Fixed it. Nice peace sign emoji. Woof.

**Pull request merged**